WebContent/js/components/Setting.js

```diff
@@ -0,0 +1,117 @@
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import styled from 'styled-components';
+import { ENABLE_REDUX_LOGGER, NOTIFICATION_DURATION, getStorageItem, setStorageItem } from '../utilities/storageHelper';
+
+
+const Settings = styled.div`
+    width: 180px;
+    padding: 15px;
+    margin: 0px 0px 15px;
+    fontSize: 13px;
+    color: #39464e;
+    fontFamily: "Roboto", "Helvetica", "Arial", sans-serif;
+`;
+const H3 = styled.h3`
+    border-top-color:rgb(233, 233, 233);
+    border-top-style:solid;
+    border-top-width:1px;
+    color:rgb(57, 70, 78);
+    font-size:13px;
+    font-weight:700;
+    padding-top:8px;
+`;
+
+const SettingSection = styled.div`
+    width: 100%;
+    display: grid;
+    grid-template-columns: 100%
+`;
+
+export default class SettingForm extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            enableReduxLogger: getStorageItem(ENABLE_REDUX_LOGGER) || false,
+            notificationDuration: getStorageItem(NOTIFICATION_DURATION) || 5000,
+        };
+
+        this.mapStorageKey = new Map();
+        this.mapStorageKey.set(ENABLE_REDUX_LOGGER, 'enableReduxLogger');
+        this.mapStorageKey.set(NOTIFICATION_DURATION, 'notificationDuration');
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange = ev => {
+        let value = ev.target.value;
+        const key = this.mapStorageKey.get(ev.target.name);
+
+        if ('checked' in ev.target) {
+            value = ev.target.checked;
+        }
+
+        this.setState({
+            [key]: value,
+        });
+
+        setStorageItem(ev.target.name, value);
+    }
+
+    render() {
+        const { notificationDuration, enableReduxLogger } = this.state;
+        return (
+            <Settings>
+                <h3>Preferences</h3>
+                <form>
+                    <H3>App</H3>
+                    <SettingSection>
+                        <FormControl>
+                            <TextField
+                                select={true}
+                                label="*Notification Duration"
+                                value={notificationDuration.toString()}
+                                onChange={this.handleChange}
+                                name={NOTIFICATION_DURATION}
+                            >
+                                <MenuItem id="notification-small" key="small" value="5000" >Small</MenuItem>
+                                <MenuItem id="notification-medium" key="medium" value="10000" >Medium</MenuItem>
+                                <MenuItem id="notification-large" key="large" value="15000" >Large</MenuItem>
+                            </TextField>
+
+
+                        </FormControl>
+                    </SettingSection>
+                    <H3>Development</H3>
```

Comment:
I don't think having a section called development looks good to user, For now we could title it just Logging

Reply:
done